src/sx-design/src/Menu/__tests__/Menu.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @flow
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import { render, fireEvent } from '../../test-utils';
+import Menu from '../Menu';
+
+it('renders the menu without any issues', () => {
+  const { getByRole, getByText, getByTestId, queryByText, queryByTestId } = render(
+    <Menu>
+      <Menu.Item onClick={jest.fn()}>One</Menu.Item>
+      <Menu.Item onClick={jest.fn()}>Two</Menu.Item>
+      <Menu.Item tint="default" onClick={jest.fn()}>
+        Three
+      </Menu.Item>
+      <Menu.ItemDivider data-testid="MenuItemDivider" />
+      <Menu.Item tint="error" onClick={jest.fn()}>
+        Delete
+      </Menu.Item>
+    </Menu>,
+  );
+
+  // Everything is hidden by default (not in the DOM):
+  expect(queryByText('One')).toBeNull();
```

Comment:
What about using [toBeInTheDocument](https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument)

Reply:
I was not aware of this function, thanks!

This brings a question though. I was always using something like this in my tests:

```js
expect(getByText('Click me!')).toBeDefined();
```

Are you saying it's better to use this?

```js
expect(getByText('Click me!')).toBeInTheDocument();
```

And of course, if yes, do you know why? What is the difference? 🤔 

Thanks!

Reply:
I didn't dive into the specifics, but at least from my pov it reads better. 

There might be some other benefit/edge case it covers 🤷‍♂️